scripts/install

```diff
@@ -226,7 +226,8 @@ ynh_print_info "Configuring SSOwat..."
 
 # Make app public
 ynh_app_setting_set $app unprotected_uris "/"
-ynh_app_setting_set $app skipped_regex "^[^/]*/[Aa]uto[Dd]iscover.*"
+domain_regex=$(echo "$domain" | sed 's@-@.@g')
```

Comment:
You have to add path_url too, like this:
https://github.com/YunoHost-Apps/lutim_ynh/blob/ba066067459743b9574d06bccdb858062a298ccb/scripts/install#L198-L204

Reply:
Not sure it's requested:

`ynh_app_setting_set $app unprotected_uris "/"` manage `path_url="/Microsoft-Server-ActiveSync"`

and 
`ynh_app_setting_set $app skipped_regex "$domain_regex/[Aa]uto[Dd]iscover/.*"`
manage all `/autodiscover/autodiscover.xml`

Reply:
This will skip only: `$domain_regex/[Aa]uto[Dd]iscover/.*`
But the app is accessible with `$domain_regex$path_url`, not only `$domain_regex`

Reply:
Ok, misread the code, your regex is ok :-)

Reply:
Thanks for the review :)